.github/pull_request_template.md

```diff
@@ -1,13 +1,25 @@
 <!--
 
-Describe in detail the changes you are proposing, and the rationale.
+Describe in detail the changes you are proposing, and the rationale. If practical, describe how to manually test the change; this isn't always possible, but it makes PRs easier and faster to review.
 
 See the contributing guide:
 
 https://github.com/hashicorp/terraform/blob/main/.github/CONTRIBUTING.md
 
 -->
 
```

Comment:
```suggestion
Fixes #

<!-- Link all GitHub issues fixed by this PR, and add references to prior related PRs. -->
```

Bring this up to the top since it's the most important thing.